docs/application/native/guides/ui/efl/wearable/component-circle-datetime.md

```diff
@@ -56,7 +56,7 @@ To configure the circle properties of the circle datetime:
 
   To change the item properties, use the `eext_circle_object_item_XXX()` functions.
 
-For more information, see [Circle Object](component-circ-object-w.md) and the [Efl Extension Circle Object](../../../../../org.tizen.native.wearable.apireference/group__CAPI__EFL__EXTENSION__CIRCLE__OBJECT__MODULE.html) API.
+For more information, see [Circle Object](component-circ-object.md) and the [Efl Extension Circle Object](../../../../../org.tizen.native.wearable.apireference/group__CAPI__EFL__EXTENSION__CIRCLE__OBJECT__MODULE.html) API.
```

Comment:
Broken link : `component-circ-object.md` should be changed into `component-circle-object.md`.

Reply:
done